### ISSUE TITLE
how about adding Angularjs to Nodered for more flexible and dynamic template

### ISSUE THREAD

**Issue by leonardean** [node-red/node-red#761]
Using jQuery like that is quite painful when creating Nodes

**Comment by knolleary** [comment]
Hi, fyi for general feature requests/discussions, as described in our contributing guidelines, we prefer to use the [mailing list](https://groups.google.com/forum/#!forum/node-red) and keep this list for real issues.

On this topic, any change would be a considerable piece of work in order to ensure a smooth co-existence of existing node templates and whatever 'new' thing was added.

We also have to be considerate of our developer community - why would we add angularjs rather than one of the many other toolkits? I can't say I've used enough angular to have an opinion on it.

jQuery is at least a stable common baseline that is accessible to most without getting too drawn into any one particular toolkit that may come and go.

So, I'm not saying we wouldn't consider it - but we would need to a more reasoned discussion around the pros and cons rather than just a distaste of jquery.
